main.py

```diff
@@ -38,6 +38,9 @@
 import telescope.utils
 
 
+MAX_THREADS_NORMAL_MODE = 18
+MAX_THREADS_BATCH_MODE = 100
```

Comment:
Two hours of my evening sunk into a minor git issue caused by committing .DS_Store.  I cleaned everything up but this showed. Not completely sure.